### PR TITLE
Summarise top/bottom samples

### PR DESCRIPTION
**Pull request by charyeezy**
Core changes - 
- Added `summarize_closest_to_center` which performs `list_closest_to_center` with default `page_size=3` on `select_fields`.
- - Added `summarize_furthest_from_center` which performs `furthest_from_center` with default `page_size=3` on `select_fields`.
- TransformerLMSummarizer class which instantiate seq2seq Transformer model (`bart` or `t5` base) to generate summary from the joined phrases and output that to this object.

eg.
```python
summarised_center_docs = cluster_ops[t].summarize_closest_to_center(
    select_fields=[t],
    # page_size = 3 (default)
)
```

```json
{'results': {'cluster-0': [{'actionable-recommendation': 'All membrane systems used under balcony pedestal pavers are to have an appropriate level of UV stabilisation and will be reviewed by 3rd Party Waterproof consultant. Ensure all mechanical penetrations at roof level are a minimum 250mm apart to allow for 100mm hob, 50mm space,100mm hob for complaint waterproofing.'}],
 'cluster-1': [{'actionable-recommendation': "CNN.com will feature iReporter photos in a weekly Travel Snapshots gallery. Visit CNN.com/Travel next week for a new gallery of snapshots of the world's best travel destinations. Click here for a gallery of photos of the best destinations in the U.S."}],
 'cluster-10': [{'actionable-recommendation': "Review opportunities to streamline reporting and approvals where possible. Streamlined process for defecting that allows for Construction / CRM's / PSI's to be conducted for thorough defecting reviews. Review opportunities for construction/CRM's/PSI's reviews are clear."}],
 'cluster-11': [{'actionable-recommendation': 'More steel and slab footing extensions were_x000D_needed in the structural design to maintain a ridged structure. Assessment needed in coming years of whether the quality of finish provided by_x, and in later stages the Hebel panels is upheld. More steel & slab footing extension were_X000D\xa0needed in'}],
}
```

Additional cleanup changes - 
- renamed '-' to '_' for consistency between `ClusterOps` and autocluster `alias`
- cleaned ReadMe (fixing client instantiation project, key, region to token)
- added additional  `cluster_properties_filter: Optional[Dict] = {}` param from [API](https://docs.relevance.ai/reference/listclosesttocentroids) `list_closest_to_center` and `list_furthest_from_center`

### CODE REVIEW

**Comment by JackyKoh** [comment]
@charyeezy can we split this pr into 2, atm i see there is a renaming pr and a summarize pr.

**Comment by JackyKoh** [comment]
the summarizer PR looks good to me @charyeezy

File: relevanceai/operations/cluster/centroids.py [hunk 1]
```diff
@@ -1,3 +1,317 @@
+<<<<<<< HEAD:relevanceai/workflows/cluster_ops/centroids.py
```
Comment (boba-and-beer) [inline relevanceai/operations/cluster/centroids.py:1]: leftover

File: tests/globals/clusterers/minibatch_clusterer.py [hunk 2]
```diff
@@ -40,6 +49,14 @@ def minibatch_subclusterer(test_dataset: Dataset):
 
 
 @pytest.fixture(scope="function")
+<<<<<<< HEAD
```
Comment (boba-and-beer) [inline tests/globals/clusterers/minibatch_clusterer.py:52]: leftover

**Comment by charyeezy** [comment]
Going to close and re-open on branch post-refactor.

**Pull request closed without merging**